addusers.py

```diff
@@ -102,66 +95,44 @@ def get_url(self, token):
 
 class Openstack:
 
-    def __init__(self, uname, password, tname, auth_url, nova_version):
-        self.keystone = client.Client(username=uname,
-                                      password=password,
-                                      tenant_name=tname,
-                                      auth_url=auth_url)
-        self.nova = novaclient.Client(nova_version,
-                                      uname,
-                                      password,
-                                      tname,
-                                      auth_url)
-
-        """
-        region_name is passed to neutronclient to avoid this warning: 
-             keystoneclient/service_catalog.py:196: UserWarning: Providing 
-             attr without filter_value to get_urls() is deprecated as of 
-             the 1.7.0 release and may be removed in the 2.0.0 release." 
-             Either both should be provided or neither should be provided.
-        Liberty neutronclient hard-codes passing the attr 'region' so we
-        must supply the filter_value region_name.  This issue is fixed in
-        Mitaka neutronclient.
-        """
-        self.neutron = neutronclient.Client(username=uname,
-                                            password=password,
-                                            tenant_name=tname,
-                                            auth_url=auth_url,
-                                            region_name=region_name)
-        self.cinder = cinderclient.Client(uname,
-                                          password,
-                                          tname,
-                                          auth_url)
+    def __init__(self, session, nova_version, setpass_url):
+        self.keystone = client.Client(session=session)
+        self.nova = novaclient.Client(nova_version, session=session)
+        self.neutron = neutronclient.Client(session=session)
+        self.cinder = cinderclient.Client(session=session)
```

Comment:
I think we do not need it because we explicitly import cinderclient v2:
https://github.com/CCI-MOC/moc-openstack-tools/pull/8/files#diff-f2b89bcfe38a6f93d893d739fb04b306R43

but, this raises a good point - it might be better to just import client and specify the version